host_vars/deploy/main.yml

```diff
@@ -28,6 +28,7 @@ ngi_pipeline_path: "{{ proj_root }}/{{ uppmax_project }}/private"
 ngi_pipeline_nobackup: "{{ proj_root }}/{{ uppmax_project }}/nobackup"
 ngi_pipeline_workdir: "{{ ngi_pipeline_nobackup }}/NGI"
 ngi_log_path: "{{ ngi_pipeline_path }}/log"
+genomes_basedir: "{{ deployment_root }}/common/data/uppnex/"
```

Comment:
I would guess there could be confusions between `--igenomes_base`, `--gennomes_base`(sarek only) and this new `genomes_basedir` which are all different...

Reply:
agreed